models/community.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const CommunitySchema = new mongoose.Schema({
+  name: { type: String, unique: true, maxlength: 20, trim: true, required: true },
+  creator_id: { type: String },
+  group: { type: String }, // group Id or public name
+  channel1: { type: String }, // channel 1 Id or public name, usually buy, if only one channel is used then buy/sell
+  channel2: { type: String }, // channel 2 Id or public name, usually sell
+  channel3: { type: String }, // channel 3 Id or public name, we use this channel to admin the community
```

Comment:
`admin_channel` or `dispute_channel`?

Reply:
that's right we better talk about dispute channel